docs/the-new-architecture/pillars-fabric-components.md

```diff
@@ -96,8 +96,7 @@ import type { HostComponent } from 'react-native';
 import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
 
 export interface NativeProps extends ViewProps {
-  ...ViewProps,
-  text: string | null | undefined,
+  text?: string;
```

Comment:
@cipolleschi made the `text` property optional, verified that codegen works with it